src/io/flutter/logging/FlutterLogView.java

```diff
@@ -287,7 +292,7 @@ public void actionPerformed(AnActionEvent e) {
         if (paths != null) {
           final Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
           final StringBuilder sb = new StringBuilder();
-          for (final TreePath path: paths) {
+          for (final TreePath path : paths) {
```

Comment:
It looks like our formatting settings don't agree...  When I format it removes the space before `:`.  Does it not for you?

Reply:
Look likes the coding style is different.
How did you setting code style ?

Reply:
Interesting.  I'm using our project-level settings.  (And I don't believe I modified them?)

@devoncarew or @stevemessick : what happens when you format

```java
        for (String s : new ArrayList<String>()) {

        }
```

in the flutter-intellij project?

Reply:
My Intellij IDE will keep space before `:`.

Reply:
Thanks!  I think this is an issue w/ defaults in EAP 2018.2.

Reply:
Oh, I got it.

Reply:
Looks like the final decision is that IntelliJ 2018.2 will be adding spaces before colons in for-each loops by default. It's configurable of course. See today's [blog post](https://blog.jetbrains.com/idea/2018/07/intellij-idea-2018-2-beta-2-is-available/):
`Starting with this build IntelliJ IDEA will, by default, add spaces before colons in a for-each loop.`

Reply:
Thanks for following up @alexander-doroshko !